include/outputs

```diff
@@ -172,13 +172,13 @@ textFail(){
     # Resource value is the second field of line included in whitelist divided by :
     resource_value=$(awk -F ":" '{print $2}' <<< $excluded_item)
     # Checked value is the second field of log message divided by spaces
-    checked_value=$(awk '{print $2}' <<< $1)
+    checked_value=$1
```

Comment:
Please link `$1` with local variable with a proper name to better follow the execution flow.

Also, quote variables --> `"${varName}"`